### DISCUSSION TITLE
Target class [Spatie\Permission\Middlewares\RoleMiddleware] does not exist. in laravel 10

### DISCUSSION THREAD

**Discussion by yahiabentale** [spatie/laravel-permission#2536] [Q&A]
Target class [Spatie\Permission\Middlewares\RoleMiddleware] does not exist. in laravel 10

**Comment by drbyte** [comment]
Sounds like you're using v6 with code that relied on v5 namespacings. Please look at the upgrade documentation for the things to address in your app when upgrading.
